client/state/directly/reducer.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	DIRECTLY_INITIALIZE
+} from 'state/action-types';
+import { createReducer } from 'state/utils';
+
+export const isInitialized = createReducer( false, {
+	[ DIRECTLY_INITIALIZE ]: () => true,
+} );
```

Comment:
> It's a little unfair to claim that the utility creates heavy reducers though; by it's implementation, it's effectively a hasOwnProperty check for an unhandled action type.

thanks @aduth - I think I've had the wrong impression of it. still, I find it obscure because I never can remember what all it does.

marginally related: @gwwar and I have spoken about the idea of making persistence _opt-in_ only and with schema validation. the idea is that we could add some `.schema` property on a reducer if we want it to persist and without that it just wouldn't save.